style.css

```diff
@@ -1,18 +1,81 @@
+.position__nav {
+  position: fixed;
+  top: 0;
+  right: 0;
+  z-index: 1;
+}
+
 .toolbar {
   display: flex;
   justify-content: space-between;
   width: 100%;
+  background-color: #fff;
+}
+
+.toolbar__open {
+  display: grid;
+  width: 100%;
+  background-color: #fff;
+}
+
+.toolbar__open__row1 {
+  grid-row: 1/2;
+}
+
+.toolbar__open__row2 {
+  grid-row: 2/3;
+  grid-column: 1/3;
+  justify-self: end;
+  margin: 0 20%;
+}
+
+.toolbar__open__row3 {
+  grid-row: 3/4;
+  grid-column: 1/3;
+  justify-self: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 2.8vh;
+  line-height: 300%;
+  color: #000;
+}
+
+.toolbar__open__row4 {
+  grid-row: 4/5;
+  grid-column: 1/3;
+  width: 100%;
 }
 
 .corner__geometry {
   margin: 0;
 }
 
+.menu__button__align {
+  justify-self: end;
+}
+
 .nav__icon {
   width: 20px;
   height: 2px;
 }
 
+#hamburger {
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  border: 0;
+  font-size: xx-large;
+}
+
+.opened__menu {
+  display: none;
+}
+
+main {
```

Comment:
- [ ] Kindly make all styling changes which is pointed out [here](https://www.loom.com/share/54141dfac67a4280b96ffca051c286f8).